capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name
+  end
+end
```

Comment:
Kindly make sure that the `CapitalizeDecorator` capitalizes the first letter of the given name.

suggestion:

```ruby
...
    def correct_name
      @nameable.correct_name.capitalize
    end
...
```